queries/landuse.jinja2

```diff
@@ -29,19 +52,27 @@ WHERE
 UNION ALL
 
 SELECT
-    name,
-    {{ ne_landuse_cols('park or protected land') }}
+  name,
+  tags->'protect_class' AS protect_class,
+  way_area::bigint AS area,
+  mz_calculate_landuse_kind("landuse", "leisure", "natural", "highway", "aeroway", "amenity", "tourism", "man_made", "power", "boundary") AS kind,
+  'openstreetmap.org' AS source,
+  {% filter geometry %}way{% endfilter %} AS __geometry__,
+  osm_id AS __id__
 FROM
-    ne_10m_parks_and_protected_lands
+  planet_osm_polygon
 WHERE
-    {{ bounds|bbox_filter('the_geom') }}
+  {{ bounds|bbox_filter('way') }}
+  AND boundary IN ('national_park', 'protected_area')
+  AND mz_one_pixel_zoom(way_area) <= {{ zoom }}
```

Comment:
Ah, this is something slightly different; rather than the number of pixels it'll be at any zoom, this function returns the (fractional) zoom at which the feature will be one pixel in size. This allows us to adjust it by changing the comparison (e.g: `<= {{ zoom }} - 1` if we want it to be at least 4 square pixels). I would call it `mz_zoom_at_which_this_feature_is_one_square_pixel`, but that doesn't exactly roll off the tongue. :wink: